make/run_make.mk

```diff
@@ -38,5 +38,5 @@ plain smp frag smp_frag:
 	$(make_verbose)$(MAKE) -f $(TARGET)/Makefile FLAVOR=$@
 
 clean generate depend docs release release_spec release_docs release_docs_spec \
-  tests release_tests release_tests_spec static_lib xmllint:
+  tests release_tests release_tests_spec static_lib xmllint clang_format compdb:
```

Comment:
4 lines above there is a reference to four different flavors: plain, smp, frag, smp_frag. Are those additional flavors compared to `emu | asm` or are the flavors above outdated? FWIW, smp_frag doesn't seem to appear anywhere in the codebase and `plain` is referred to in three or four places.

Reply:
They are all outdated. `plain` and `smp` were the non-SMP and SMP runtime systems, respectively. `frag` and `smp_frag` are from the time when we first implemented literals (OTP R11B).

Reply:
I've removed the old flavors and added the new